### PR TITLE
Add QingCloud provider for salt-cloud

### PR DESCRIPTION
**Pull request by vinta**
QingCloud is one of most popular IaaS in China. This pull request try to integrate QingCloud with SaltStack.

https://www.qingcloud.com/

### CODE REVIEW

**Comment by techhat** [comment]
Hey @vinta, this looks really good. There are some lint errors that need to be dealt with... in particular, would you please remove the docstrings with the ASCII art text?

**Comment by vinta** [comment]
@techhat Sure! I will fix these.

File: salt/cloud/clouds/qingcloud.py [hunk 1]
```diff
@@ -0,0 +1,845 @@
+# -*- coding: utf-8 -*-
+'''
+QingCloud Cloud Module
+======================
+
+.. versionadded:: 2015.2
```
Comment (rallytime) [inline salt/cloud/clouds/qingcloud.py]: This should actually be a `.. versionadded:: Beryllium` since this is a new feature going into develop.

**Comment by techhat** [comment]
@vinta, pending fixing the version as pointed out by @rallytime, I'm good with this getting merged. Thanks for doing this!

**Comment by rallytime** [comment]
Thanks!

**Pull request merged**